src/Aspire.Hosting/DistributedApplicationBuilder.cs

```diff
@@ -313,10 +313,7 @@ private void ConfigureHealthChecks()
             {
                 if (ExecutionContext.IsRunMode)
```

Comment:
Delete this completely.

Reply:
Need to keep it there for publish mode. But I've removed the other side of the branch.

Reply:
Why? Who is doing health checks?